homeassistant/components/amcrest/binary_sensor.py

```diff
@@ -39,7 +39,7 @@
 class AmcrestSensorEntityDescription(BinarySensorEntityDescription):
     """Describe Amcrest sensor entity."""
 
-    event_code: str | None = None
+    event_codes: list[str] | None = None
```

Comment:
Should it be a set instead? The codes should be unique, right?

Reply:
Good idea, changed over to a set.